packages/@aws-cdk/aws-pipes-enrichments-alpha/test/__snapshots__/stepfunctions.test.ts.snap

```diff
@@ -10,15 +10,7 @@ exports[`stepfunctions should grant pipe role invoke access 1`] = `
             "Action": "sts:AssumeRole",
             "Effect": "Allow",
             "Principal": {
-              "Service": {
-                "Fn::FindInMap": [
-                  "ServiceprincipalMap",
-                  {
-                    "Ref": "AWS::Region",
-                  },
-                  "states",
-                ],
-              },
+              "Service": "states.amazonaws.com",
```

Comment:
I believe this service principal should have the region in it, i.e. `states.us-west-1.amazonaws.com`. This could be right since the principal is sans region some of the time, but looking through the regional maps we had, a majority of the regions are included in the principal.

Reply:
Or at least it should not be static if it can be either. The test should be environment agnostic and be a mapping, correct?

Reply:
The only context in which any region should be included is in opt in regions when the permissions are cross region. This is not that case.